spec/features/posting_peep_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'posting peep' do
+  scenario 'A user can post a peep to Chitter' do
+    visit('/chitter/new')
+    fill_in('peep', with: 'Hello from user1')
+    click_button('Post')
+
+    expect(page).to have_content 'Hello from user1'
```

Comment:
could also use this test to specify how you want to see the peep - ie. not in an array